exercises/G-conditionals-2/exercise-3.js

```diff
@@ -8,7 +8,17 @@
   - Otherwise the grade is "F"
 */
 
-function calculateGrade(mark) {}
+function calculateGrade(mark) {
+  if (mark >= 80){
+    return "A";
+  } else if (mark < 80 && mark > 60){
+    return "B";
+  } else if (mark < 60 && mark > 50){
+    return "C";
+  } else {
+    return "F"
+  }
+}
```

Comment:
This works, but you could simplify the code. If the mark is greater or equal to 80, then you code will exit at line 13. So you don't need to do the check
mark < 80
on line 14. If the mark must be less than 80, or your code would already have exited.
Similarly, you can simplify line 16